lib/core/utils/get-frame-contexts.js

```diff
@@ -0,0 +1,11 @@
+import Context from '../base/context'
+import getAncestry from './get-ancestry'
+
+export default function getFrameContexts(context) {
+  const { frames } = new Context(context);
```

Comment:
This creates `axe._tree`, which is going to slow stuff down. Can we avoid that? And if not, how do we avoid either a tree that is outdated by the time axe.runPartial is called, or generating the tree twice.

Reply:
Do we want to fix this in this pr or another one?

Reply:
Nope, we can let Context build axe._tree. I've accounted for it in the updated proposal.